src/python/WMComponent/DBS3Buffer/MySQL/NewSubscription.py

```diff
@@ -32,13 +32,24 @@ def execute(self, datasetID, subscriptionInfo,
         Execute the query and retrieve the subscriptions
         """
         binds = []
+
+        # DeleteFromSource is not supported for move subscriptions
+        delete_blocks = None
+        if subscriptionInfo['CustodialSubType'] == 'Move':
+                isMove = 1
+        else:
+                isMove = 0
+                if subscriptionInfo.get('DeleteFromSource', False):
```

Comment:
Yes, it's part of the subscription information that is set in the spec. I can use it as-is from the Tier0 as I construct this myself in code. If we want to use it for normal requests some more code will be needed. But for now this just adds the capability, if not set (which is the default) it won't do anything.